client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -395,6 +407,36 @@ class RequiredPluginsInstallView extends Component {
 		return TIME_TO_PLUGIN_INSTALLATION;
 	};
 
+	renderContactSupport() {
+		const { translate, wporg } = this.props;
+		const { workingOn } = this.state;
+		const plugin = getPlugin( wporg, workingOn );
+		return (
+			<div className="dashboard__setup-wrapper setup__wrapper">
+				<div className="card dashboard__plugins-install-view">
+					<SetupHeader
+						imageSource={ '/calypso/images/extensions/woocommerce/woocommerce-store-creation.svg' }
+						imageWidth={ 160 }
+						title={ translate( "We can't update your store" ) }
+						subtitle={ translate(
+							"Your store is missing some required plugins. We can't fix this automatically due to " +
+								'a problem with the {{b}}%(plugin)s{{/b}} plugin. {{br/}}{{br/}}' +
```

Comment:
I would recommend to not include the double `<br/>` in the translatable string. Could you split the translate calls into two that only contain the translatable text? Thanks!